web-api/openapi/front_office.yaml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.3
 info:
   title: Front Office API
-  version: 1.0.0-rc6
+  version: 1.0.0-rc7
```

Comment:
:wrench: `/users/.../personalized_documents` spec must be updated with the new parameter.